v1.0/image.schema.json

```diff
@@ -9,30 +9,24 @@
   ],
   "properties": {
     "coordinateSpace": {      
-      "definition": "Coordinate space the image exists in.",
+      "_instruction": "Add the coordinate space this image exists in.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/core/CoordinateSpace"
       ]
     },
     "definedIn":{     
-      "description": "File in which the image is defined.",
+      "_instruction": "Add the file in which this image is defined in.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/core/FileInstance"
       ]
-    },
-    "unit": {
-      "description": "A determinate quantity adopted as a standard of the measurement.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/controlledTerms/Unit"
-      ]
-    },    
+    },   
     "voxelSize": {
       "type": "array",
-      "description": "Corresponding size of one voxel (3D) or pixel (2D, with z = 0) in the physical space, defined in the unit of this image.",
-      "items": {
-        "type": "number"
-      },
-      "minItems": 3,
+      "_instruction": "Add two or three values with units defined that describe the size of one pixel (2D, with 2 values) or voxel (3D, with 3 values) in the physical space.",
```

Comment:
Add three quantitative values that describe the voxel size used in this image (for 2D: Z = 0)